.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-- 6.2.0
-- 10
+- 8.10.0
```

Comment:
maybe use `lts` here.

Reply:
Hmm. Based on the docs is it `lts` or `lts/*`? I guess I'll try one and see what happens...